SPECS/libxml2/libxml2.spec

```diff
@@ -1,5 +1,3 @@
-%{!?python2_sitelib: %define python2_sitelib %(python2 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
-%{!?python3_sitelib: %define python3_sitelib %(python3 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
```

Comment:
Is this already defined in our macros and that's why we can remove it?

Reply:
Yes, this is provides as part of `macros.python3` in the `mariner-rpm-macros` package.